common/t5common/jira/check_jira.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+import asyncio
+import json
+import os
+from os.path import abspath, relpath
+import re
+import sys
+import subprocess
+import time
+
+import yaml
+
+from .connector import JiraConnector
+from .database import DBConnector
+from .utils import load_config, get_job_env, open_wf_file
+from ..utils import get_logger, read_token
+
+
+logger = get_logger()
+
+
+def format_query(config):
+    return 'project = {project} AND status = "{new_status}"'.format(**config)
+
+
```

Comment:
Could you add docstrings for the different functions in `check_jira.py`

Reply:
Done.